internal/lint/spy.go

```diff
@@ -0,0 +1,191 @@
+package lint
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"golang.org/x/mod/modfile"
+)
+
+type spyKey struct{}
+
+type spy struct {
```

Comment:
I would prefer a name that reflects the information contained and/or the purpose of this strucutre. 
Is this a linter context?

Reply:
In the meantime I renamed it to `state`